src/Consoles/Screen.cs

```diff
@@ -68,7 +68,17 @@ public Screen(int width, int height)
                             new TestItem(AllowedItem.Single), new TestItem(AllowedItem.Single)
                         });
                     MenuConsole.OpenChest(chest);
-                };
+                };
+
+            // Nie wiem czy nie mamy buga bo w ten spos�b nie wy?wietlaj? si? najpierw poprzednie wiadomo?ci
+            // chyba ?e zawsze b?dziemy u?ywa? PrintMessageAndWait ale to podczas walki byloby slabe

```

Comment:
Ok, już czaję. Widzisz, z twojego komentarza to niekoniecznie wynikało 😉
Generalnie w naszej rozgrywce nie ma na razie sytuacji (lub też nie powinno być) w której dwa obiekty będą na raz wypisywać coś na MessageConsole, ale zacząłem myśleć o czymś takim jak PrintMessageWithTimeout(), który będzie zajmował mejsce w kolejce oczekiwania, a po określonym czasie zniknie.

Normalnie będziemy odpalali Message/Menu z metody ProcessKeyboard, więc z moim PR #54 MainConsole nie odpali Update() dopóki nie skończy sie akcja aktywnej konsoli.

Te testowe rzeczy ze Screena będziemy chcieli przenieść do testowego Roomu.